onyo/commands/get.py

```diff
@@ -127,14 +127,24 @@ def get(args: argparse.Namespace, opdir: str) -> None:
     except OnyoInvalidRepoError:
         sys.exit(1)
 
-    # validate arguments
-    paths = set(Path(p) for p in args.path) or {Path('.')}
-    invalid_paths = {p for p in paths if not p.exists()}
-    paths -= invalid_paths
+    # validate path arguments
+    paths = set()
+    invalid_paths = set()
+    for path in args.path:
+        try:
+            onyo_path = repo.sanitize_path(path)
+            if onyo_path.exists():
+                paths.add(onyo_path)
+            else:
+                invalid_paths.add(path)
+        except ValueError:
+            invalid_paths.add(path)
     if any(invalid_paths):
```

Comment:
I know, it's not actually part of the PR, but just stumbled upon it.
The `any` construct does nothing here.

```suggestion
    if invalid_paths:
```